hack/jenkins/e2e.sh

```diff
@@ -521,7 +528,7 @@ case ${JOB_NAME} in
     : ${E2E_NETWORK:="gke-upgrade"}
     : ${E2E_OPT:="--check_version_skew=false"}
     : ${JENKINS_FORCE_GET_TARS:=y}
-    # Run release (old) e2es, not ci (new)
+    # Run old e2es
```

Comment:
Please keep your terminology consistent.  Above you used 'latest', here you use 'old'.  Perhaps standardise on RELEASE and HEAD?